src/Codeception/Command/Run.php

```diff
@@ -27,9 +27,9 @@
  *
  * * `codecept run -v`:
  * * `codecept run --steps`: print step-by-step execution
- * * `codecept run -vv`:
- * * `codecept run --debug`: print steps and debug information
- * * `codecept run -vvv`: print internal debug information
```

Comment:
What is `internal debug information`? I have no idea, so I doubt that anyone else knows what it is too.

As far as I know, the main (and only?) difference is that `-vvv` displays full stack trace in error output.

Reply:
I've taken the info from https://github.com/Codeception/Codeception/discussions/6081#discussioncomment-273324
Further down, I've called it "Codeception-internal debug" - is that better? The idea is to give users a hint that `vvv` is just for searching bugs inside Codeception (isn't it?)

Reply:
Probably it is correct.  
The difference is that in lower levels of verbosity PHPUnit and Codeception classes are removed from displayed stack trace.

Reply:
So merge it as-is? Or change something?